<?xml version="1.0"?>
<sdf version="1.6">
  <model name="uav_dummy">
    <static>false</static>
    <link name="base_link">
      <kinematic>true</kinematic>
      <gravity>false</gravity>

      <pose>0 0 0.2 0 0 0</pose>

      <inertial>
        <mass>1.0</mass>
        <inertia>
          <ixx>0.01</ixx><iyy>0.01</iyy><izz>0.02</izz>
        </inertia>
      </inertial>

      <collision name="collision">
        <geometry>
          <box><size>0.5 0.5 0.15</size></box>
        </geometry>
      </collision>

      <visual name="visual">
        <geometry>
          <box><size>0.5 0.5 0.15</size></box>
        </geometry>
        <material>
          <ambient>0 0.6 1 1</ambient>
          <diffuse>0 0.6 1 1</diffuse>
        </material>
      </visual>
    </link>
  </model>
</sdf>

